netlify.toml

```diff
@@ -1,5 +1,10 @@
 [build.environment]
 NODE_VERSION = "18"
+NETLIFY_NEXT_PLUGIN_SKIP = "true"
+
+[build]
+command = "echo 'deploying downtime page...'"
+publish = "public"
```

Comment:
No build step. We just want to publish the `public` folder that houses downtime.html.